packages/app-desktop/gui/Sidebar/Sidebar.tsx

```diff
@@ -665,6 +665,9 @@ class SidebarComponent extends React.Component<Props, State> {
 		if (this.props.folders.length) {
 			const allNotesSelected = this.props.notesParentType === 'SmartFilter' && this.props.selectedSmartFilterId === ALL_NOTES_FILTER_ID;
 			const result = shared.renderFolders(this.props, this.renderFolderItem.bind(this));
```

Comment:
is it possible to do in the shared part? this way it will be consistent between desktop and mobile (assuming shared means what I think it means)

Reply:
Yes, in fact [`models/Folder.ts`](https://github.com/laurent22/joplin/blob/8e789ee2eea693c6fa2c492acc526ecb7e96daec/packages/lib/models/Folder.ts#L543) exposes a sort function which is where this update should have happened.

Reply:
I was not aware about it, I will change it right away